src/singletons/Theme.cpp

```diff
@@ -25,15 +25,21 @@ namespace {
 using namespace chatterino;
 using namespace literals;
 
-void parseInto(const QJsonObject &obj, QLatin1String key, QColor &color)
+void parseInto(const QJsonObject &obj, const QJsonObject &fallbackObj,
+               QLatin1String key, QColor &color)
 {
-    const auto &jsonValue = obj[key];
+    auto jsonValue = obj[key];
     if (!jsonValue.isString()) [[unlikely]]
     {
-        qCWarning(chatterinoTheme) << key
-                                   << "was expected but not found in the "
-                                      "current theme - using previous value.";
-        return;
+        jsonValue = fallbackObj[key];
```

Comment:
I can't comment there, but there's also a case where the color is invalid (below) which should be handled too.

Reply:
Good catch - resolved by [`e748cf8` (#5198)](https://github.com/Chatterino/chatterino2/pull/5198/commits/e748cf84be5fe12812eecd3408f1a7bca3c6ef33)